tools/close_by_signature.py

```diff
@@ -268,6 +265,28 @@ def link_issue_to_root_issue(jira, issue, root_issue, dry_run_stdout):
         )
 
 
+TARGET_TRANSITION_NAME = 'Closed'
+
+
+def close_issue(jira, issue):
+    transition_id = get_target_transition_id(jira, issue, TARGET_TRANSITION_NAME)
+    if not transition_id:
+        return
+
+    jira.transition_issue(issue, transition_id)
+
+
+def get_target_transition_id(jira, issue, target_name):
+    for transition in jira.transitions(issue):
+        assert 'name' in transition and 'id' in transition
+
+        if transition['name'] == target_name:
```

Comment:
Why do we need to find the transiion id each time? Why not just add it as a constant here?

Reply:
as I understood, transition ids can be unique per workflow, not necessarily all tickets share the same workflow:

https://community.atlassian.com/t5/Jira-Core-Server-questions/Is-there-is-a-list-of-all-available-transition-IDs/qaq-p/406492